From cb2eed524f43ba74572799374ce0425798f67940 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <aalonso@freescale.com>
Date: Wed, 18 Feb 2015 17:48:04 -0600
Subject: [PATCH] MLK-10370: imx7d-12x12-arm2 imximage: update dcd table for
 lpddr3

* Update DCD table for lpddr3 @400Mhz
* Boot kernel linux and run memtester for memory stress
  memtester 1G 100000

Signedoff-by: Ye.Li <B37916@freescale.com>
Signed-off-by: Adrian Alonso <aalonso@freescale.com>
(cherry picked from commit 7cbab5830d486733a691be104cbc2be494b00776)
Signed-off-by: Peng Fan <Peng.Fan@freescale.com>
---
 .../freescale/mx7d_12x12_lpddr3_arm2/imximage.cfg  | 36 +++++++++++++---------
 1 file changed, 22 insertions(+), 14 deletions(-)

diff --git a/board/freescale/mx7d_12x12_lpddr3_arm2/imximage.cfg b/board/freescale/mx7d_12x12_lpddr3_arm2/imximage.cfg
index 2536309..51e7c09 100644
--- a/board/freescale/mx7d_12x12_lpddr3_arm2/imximage.cfg
+++ b/board/freescale/mx7d_12x12_lpddr3_arm2/imximage.cfg
@@ -58,11 +58,16 @@ DATA 4 0x30340004 0x4F400005
 
 DATA 4 0x30391000 0x00000002
 DATA 4 0x307a0000 0x03040008
-DATA 4 0x307a0064 0x00210038
+
+DATA 4 0x307a01a0 0x80400003
+DATA 4 0x307a01a4 0x00100020
+DATA 4 0x307a01a8 0x80100004
+
+DATA 4 0x307a0064 0x00200038
 DATA 4 0x307a0490 0x00000001
 DATA 4 0x307a00d0 0x00350001
 DATA 4 0x307a00dc 0x00c3000a
-DATA 4 0x307a00e0 0x00020000
+DATA 4 0x307a00e0 0x00010000
 DATA 4 0x307a00e4 0x00110006
 DATA 4 0x307a00f4 0x0000033f
 DATA 4 0x307a0100 0x0a0e110b
@@ -77,14 +82,11 @@ DATA 4 0x307a0180 0x00600018
 DATA 4 0x307a0184 0x00e00100
 DATA 4 0x307a0190 0x02098205
 DATA 4 0x307a0194 0x00060303
-DATA 4 0x307a01a0 0x80400003
-DATA 4 0x307a01a4 0x00001020
-DATA 4 0x307a01a8 0x80100004
 
 DATA 4 0x307a0200 0x00000016
 DATA 4 0x307a0204 0x00171717
 DATA 4 0x307a0214 0x05050505
-DATA 4 0x307a0218 0x00050505
+DATA 4 0x307a0218 0x0f0f0505
 
 DATA 4 0x307a0240 0x06000601
 DATA 4 0x307a0244 0x00000000
@@ -94,20 +96,26 @@ DATA 4 0x30790004 0x10210100
 DATA 4 0x30790008 0x00010000
 DATA 4 0x30790010 0x0007080c
 DATA 4 0x3079001C 0x01010000
-DATA 4 0x3079009c 0x0db60db6
-DATA 4 0x30790018 0x0000000f
-DATA 4 0x307900c0 0x0f407304
-DATA 4 0x307900c0 0x0f447304
-DATA 4 0x307900c0 0x0f447306
+
+DATA 4 0x3079009c 0x00000d6e
+DATA 4 0x30790020 0x0a0c0a0a
+DATA 4 0x30790030 0x06060606
+DATA 4 0x30790050 0x01000008
+DATA 4 0x30790050 0x00000008
+DATA 4 0x307900c0 0x0e407304
+DATA 4 0x307900c0 0x0e447304
+DATA 4 0x307900c0 0x0e447306
 
 CHECK_DATA 4 0x307900c4 0x1
 
-DATA 4 0x307900c0 0x0f407304
+DATA 4 0x307900c0 0x0e4c7304
+DATA 4 0x307900c0 0x0e487306
+
 DATA 4 0x30384130 0x00000000
 DATA 4 0x30340020 0x00000178
 DATA 4 0x30384130 0x00000002
-DATA 4 0x30790050 0x01000018
-DATA 4 0x30790050 0x00000018
+
+DATA 4 0x30790018 0x0000000f
 
 CHECK_DATA 4 0x307a0004 0x1
 #endif
-- 
1.8.0

